File: Selection Sort/Selection_sort.py
def Selection_sort(arr):                        #Function to apply selection sort on the given aray
  for i in range(len(arr)):
    min=i
    for j in range(i+1,len(arr)):
      if(arr[j]<arr[min]):
        min=j
    if(min!=i):
      temp=arr[i]
      arr[i]=arr[min]
      arr[min]=temp

  #Printing the sorted array
  for k in range(len(arr)):
    print(arr[k], end=" ")

arr=[1,2,3,4,15,22,31,8,9,10]
Selection_sort(arr)

File: Selection Sort/Selection_sort.java
class Selection_sort
{
  public static void main(String[] args)
  {
    int arr[]={}, min, i, j;
    for(i=0; i<10; i++)
    {
      min=i;
      for(j=i+1; j<10; j++)
      {
        if(arr[j]<arr[min])
        {
          min=j;
        }
      }

      if(min!=i)
    {
      int temp=arr[i];
      arr[i]=arr[min];
      arr[min]=temp;
    } 
    }

    //Printing the sorted array
    for(int k=0; k<10; k++)
    {
      System.out.print(arr[k]+"  ");
    }
  }
}